lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/sim/XwikiController.java

```diff
@@ -181,6 +182,15 @@ public void receiveSessionScoreUpdateEvent(SessionScoreUpdateEvent event) throws
 		this.cw.receiveScoreUpdate(new ScoreRecord(event.user, event.processartifactid,
 				event.simulationsessionid, event.sessionscore));
 	}
+	
+	@Override
+	public void receiveScoreUpdateEvent(ScoreUpdateEvent event) throws LpRestException {
```

Comment:
For the moment this method is useless because it does not supersede `notifySessionScoreUpdateDemo`